### PR TITLE
add description for TaintNodesByCondition

### PR DESCRIPTION
**Pull request by tanshanshan**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5698)

### CODE REVIEW

**Comment by xiangpengzhao** [comment]
also /cc @bsalamat

File: docs/concepts/configuration/taint-and-toleration.md [hunk 1]
```diff
@@ -256,7 +256,8 @@ which matches the behavior when this feature is disabled.
 ## Taint Nodes by Condition
 
 Version 1.8 introduces an alpha feature that causes the node controller to create taints corresponding to
-Node conditions. When this feature is enabled, the scheduler does not check conditions; instead the scheduler checks taints. This assures that conditions don't affect what's scheduled onto the Node. The user can choose to ignore some of the Node's problems (represented as conditions) by adding appropriate Pod tolerations.
+Node conditions. When this feature is enabled (you can do this by including `TaintNodesByCondition=true` in `--feature-gates`, such as
```
Comment (bsalamat) [inline docs/concepts/configuration/taint-and-toleration.md]: s/`--feature-gates`/ the `--feature-gates` command line flag to the scheduler/

File: docs/concepts/configuration/taint-and-toleration.md [hunk 2]
```diff
@@ -256,7 +256,8 @@ which matches the behavior when this feature is disabled.
...
+`--feature-gates=FooBar=true,TaintNodesByCondition=true`), the scheduler does not check conditions; instead the scheduler checks taints. This assures that conditions don't affect what's scheduled onto the Node. The user can choose to ignore some of the Node's problems (represented as conditions) by adding appropriate Pod tolerations.
```
Comment (bsalamat) [inline docs/concepts/configuration/taint-and-toleration.md]: s/conditions/Node conditions/

**Comment by bsalamat** [comment]
Thanks, @tanshanshan!

/lgtm

**Pull request merged**